app/lib/publicationFiles.js

```diff
@@ -38,6 +38,7 @@ class PublicationFiles {
     const files = trailsChunked.map(chunk => {
       const newHeader = _.clone(header)
       newHeader.concern_point_hashes = this._getPointHashes(chunk)
+      if (['production','staging'].indexOf(process.env.NODE_ENV) < 0) newHeader.points_for_test = chunk
```

Comment:
Any reason we aren't just doing `process.env.NODE_ENV == "development"` here? Nothing wrong with this, just isn't the easiest to read.

Reply:
I think Jeff's rationale may be to use test data if the `NODE_ENV` environment variable was not set (e.g. use test data as long as it is not production or staging).